tests/drivers/pwm/pwm_loopback/boards/numaker_pfm_m467.overlay

```diff
@@ -0,0 +1,28 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+
+#include <zephyr/dt-bindings/pwm/pwm.h>
+
+&pinctrl {
+/* EVB's D5, D3 --> PC12, PC10 */
+	epwm1_default: epwm1_default {
+		group0 {
+			pinmux = <PC12MFP_EPWM1_CH0 0x0000>,
+					<PC10MFP_EPWM1_CH2 0x0000>;
```

Comment:
Align the `<` (sorry most editors don't get this right)

Reply:
Done